mods/d2k/rules/vehicles.yaml

```diff
@@ -415,6 +431,8 @@ raider:
 		Cost: 350
 	Tooltip:
 		Name: Raider Trike
+	Encyclopedia:
+		Description: Raiders are similar to Trikes, but the Ordos have refined their fire power, speed and armor to create a powerful and maneuverable scout. With dual 20mm cannons, Raiders are most effective against infantry and lightly armored vehicles.\n\nRaiders are vulnerable to most types of weaponry, though high caliber guns are slightly less effective.
```

Comment:
Stealth Raider is inheriting this and appearing with the same desc, apart from the prerequisites. It needs its own one.